source/Sylvan.Data.Csv.Tests/CsvDataReaderTests.cs

```diff
@@ -52,7 +52,7 @@ public void MissingFinalField2Test()
 		[Fact]
 		public async Task Simple()
 		{
-			using var csv = await CsvDataReader.CreateAsync("Data\\Simple.csv");
+			using var csv = await CsvDataReader.CreateAsync(Path.Combine("Data", "Simple.csv"));
```

Comment:
Is there a reason to not just change \ to / ? Forward slash should work everywhere, no?

Reply:
I did not know but indeed using `/` instead of `\` works under all platforms (Linux, macOS and Windows). So it's really a matter of taste. If you prefer using `/` over `Path.Combine` I can update the pull request.